application/migrations/m170408_060716_disctype_add_customview.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+use yii\db\Migration;
+
+class m170408_060716_disctype_add_customview extends Migration
+{
+    public function up()
+    {
+	$this->addColumn( \app\modules\shop\models\DiscountType::tableName(),'custom_view','VARCHAR(255) DEFAULT NULL AFTER `sort_order`');
```

Comment:
Habit of writing like this. May write
`::tableName(),'custom_view',\yii\db\Schema::TYPE_STRING . ' DEFAULT NULL');`